include/nccl_ofi.h

```diff
@@ -192,8 +192,7 @@ typedef struct nccl_ofi_connection_info {
 	nccl_net_ofi_req_t* req;
```

Comment:
Your commit message should have some explanation of what you're doing and why.  Hard to figure out from "add shim macro"

Reply:
I amended the commit message with an explanation for those unfamiliar with C++.